startup/11-temperature-controller.py

```diff
@@ -165,6 +168,10 @@ def _sts_mon(self, value, **kwargs):
 
     def set(self, val):
         self._target = val
+        if self.auto:
+            for (low, hi), heater_pos in heater_dict.items():
+                if low < pos <= hi:
+                    self.heater.set(heater_pos)
```

Comment:
you should wait on the status object returned here, or AND it with the returned status object

Reply:
feel free to push to this branch. I've already exceeded my knowledge of how things work!